plugins/flytekit-kf-pytorch/flytekitplugins/kfpytorch/task.py

```diff
@@ -189,7 +191,8 @@ def __init__(self, task_config: PyTorch, task_function: Callable, **kwargs):
             task_type_version=1,
             **kwargs,
         )
-        if self.task_config.increase_shared_mem:
+
+        if self.task_config.increase_shared_mem and (self.shared_memory is False or self.shared_memory is None):
             if self.pod_template is None:
```

Comment:
We need to actively disable this as configuring the volume mount both ways leads to a failure to create the pods due to duplicated mount paths.